keda/templates/24-metrics-apiservice.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   service:
     name: {{ .Values.operator.name }}-metrics-apiserver
-    namespace: {{ .Release.Namespace }}
+  # namespace: {{ default .Release.Namespace .Values.namespace }}

```

Comment:
it seems like we need this, shouldn't it be uncommented?

Reply:
Yeah sorry, I was hoping to solicit feedback about the general approach first but there are a number of issues to  work out and one of them is that this particular line is giving me an error when I try to install it. 

It seems like its a version mismatch with Kubernetes perhaps, but it isn't working on my local machine so I commented it out for now but yes it would need to be resolved for this to go forward.

Reply:
The other issues are the checklist items, including signed commits. I'll have to read about that a bit to make it happen.

Reply:
What was the error that you've had on this?

In terms of the commits, this should be able to help you - https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work